### PR TITLE
Add Ed25519VerificationKey2020 to key types.

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # bedrock-ssm-mongodb ChangeLog
 
+## 4.0.0 - 2021-04-TBD
+
+### Changed
+- Remove support for node 10.
```
Comment (dlongley) [inline CHANGELOG.md:6]:
```suggestion
- **BREAKING**: Remove support for node 10.
```

**Review by dlongley — APPROVED** [review]
AFAICT, LGTM.

**Review by dmitrizagidulin — APPROVED** [review]
:+1: 
Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (mattcollier) [reply CHANGELOG.md]: This was addressed.

File: test/mocha/10-asymmetric-key.js [hunk 2]
```diff
@@ -36,25 +44,79 @@ describe('asymmetric keys', () => {
...
+        const keyId = `https://example.com/kms/${uuid()}`;
+        const controller = 'https://example.com/i/foo';
+        const type = 'Ed25519VerificationKey2020';
+        const invocationTarget = {id: keyId, type, controller};
+        const result = await brSSM.generateKey(
+          {keyId, operation: {invocationTarget}});
+        should.exist(result);
+        result.should.be.an('object');
+        Object.keys(result).should.have.same.members(
+          ['id', 'publicKeyMultibase', 'type']);
```
Comment (mattcollier) [inline test/mocha/10-asymmetric-key.js:84]: `result.should.have.keys` here and on L106.

**Review by mattcollier — CHANGES_REQUESTED** [review]
TY, one nit about assertion syntax.

File: test/mocha/10-asymmetric-key.js (hunk 2 above)
Comment (JSAssassin) [reply test/mocha/10-asymmetric-key.js]: addressed here 10894aa

**Review by mattcollier — APPROVED** [review]
TY.

**Pull request merged**